src/metabase/pulse/interface.clj

```diff
@@ -0,0 +1,16 @@
+(ns metabase.pulse.interface
+  "Interface for the way parameters are handled by dashboard subscriptions, which is different in the EE and OSS
+  versions."
+  (:require [potemkin :as p]))
+
+(p/defprotocol+ SubscriptionParameters
+  "Get the parameters to be used for a dashboard subscription"
+  (the-parameters [this pulse dashboard]
+    "Return appropriate filter parameters"))
```

Comment:
This docstring is a little bit vague... it's not super clear where the parameters are coming from or what they look like. Does it return maps? Parameter IDs? Something else entirely?